### PR TITLE
Bump alpha and stable channel with the latest kops releases

### PR DESCRIPTION
**Pull request by olemarkus**

### CODE REVIEW

File: channels/alpha [hunk 1]
```diff
@@ -137,11 +137,11 @@ spec:
     requiredVersion: 1.11.10
   kopsVersions:
   - range: ">=1.25.0-alpha.1"
-    recommendedVersion: "1.25.0"
+    recommendedVersion: "1.25.2"
     #requiredVersion: 1.25.0
     kubernetesVersion: 1.25.3
   - range: ">=1.24.0-alpha.1"
-    recommendedVersion: "1.24.3"
+    recommendedVersion: "1.24.4"
```
Comment (hakman) [inline channels/alpha:144]: Time to default to 1.25.2 for older k8s versions too?
```suggestion
    recommendedVersion: "1.25.2"
```

**Review by hakman — CHANGES_REQUESTED** [review]

File: channels/alpha (hunk 1 above)
Comment (olemarkus) [reply channels/alpha:144]: Yeah. Let's do both.
Not sure about the AMI given that another one was released today.

Comment (hakman) [reply channels/alpha:144]: Let's update AMI to latest.

Comment (olemarkus) [reply channels/alpha]: Updated AMIs for alpha.

Comment (olemarkus) [reply channels/alpha]: good catch. done.

**Review by hakman — APPROVED** [review]

**Pull request merged**